### PR TITLE
Prep for touchforms removal

### PR DESCRIPTION
**Pull request by gcapalbo**
This PR:

- copies over the necessary files from the touchforms submodule into the hq repo for communicating with the formplayer api
- removes all the references to the touchforms submodule in the hq code
- makes all smsforms requests now use the formplayer backend (all active projects already are using it via the toggle)

After this it should be fine to stop running touchforms and remove the touchforms submodule from the repo, which will come in separate prs.

@wpride @snopoke

### CODE REVIEW

File: urls.py [hunk 1]
```diff
@@ -99,7 +99,6 @@
     url(r'^account/', include('corehq.apps.settings.urls')),
     url(r'^project_store(.*)$', rewrite_url),
     url(r'^exchange/', include('corehq.apps.appstore.urls')),
-    url(r'^webforms/', include('touchforms.formplayer.urls')),
```
Comment (gcapalbo) [inline urls.py]: @wpride @snopoke is it true that nothing else besides smsforms was using touchforms? i found this `/webforms/` url was still being exposed. i checked the nginx logs and for the past month it's only had 404 responses returned for bots and such

**Comment by wpride** [comment]
GIOOO 👍 🎉 🎈 💯 🔥 :tight:

**Comment by wpride** [comment]
I'm not aware of any other services using it

File: corehq/apps/formplayer_api/smsforms/api.py [hunk 2]
```diff
@@ -299,8 +299,6 @@ def formplayer_post_data_helper(d, auth, content_type, url):
     headers = {}
     headers["Content-Type"] = content_type
     headers["content-length"] = len(data)
-    # Remove Cookie header once formplayer supports mac digest header for auth
-    headers["Cookie"] = 'sessionid=%s' % settings.FORMPLAYER_INTERNAL_AUTH_KEY
```
Comment (gcapalbo) [inline corehq/apps/formplayer_api/smsforms/api.py]: @wpride when i tested locally removing this header, formplayer now returns an error saying "Invalid session id". if i put the header back and just pass in any arbitrary value for `sessionid`, i still get an error from formplayer but with an empty exception in the json output

File: urls.py (hunk 1 above)
Comment (snopoke) [reply urls.py]: I couldn't find any references to this URL or any of it's sub URLs

**Comment by gcapalbo** [comment]
@wpride when i run the tests locally now with the sessionid changes you made in https://github.com/dimagi/formplayer/pull/659, i get a different error:

```Request: http://localhost:8080/new-form raised java.lang.RuntimeException: Could not configure RestoreFactory with args [New Session Request Bean [form-url=null, postUrl= null, sessionData= SessionData: [username=abc, domain=test-domain, data={case_id_new_participant_0=ab15167dd26d401f921f68296215a842}], instanceContent=null], null]```

the test it's failing on is using a mobile worker sending sms to create a case

**Comment by wpride** [comment]
@gcapalbo https://github.com/dimagi/formplayer/pull/664

**Comment by gcapalbo** [comment]
@wpride @snopoke this is ready to go

**Review by wpride — APPROVED** [review]

**Pull request merged**